libs/core/src/primitives/form-control-footer/form-control-footer.ts

```diff
@@ -0,0 +1,59 @@
+import { property } from 'lit/decorators.js'
+import { when } from 'lit/directives/when.js'
+
+import { GdsElement } from '../../gds-element'
+import {
+  gdsCustomElement,
+  html,
+} from '../../utils/helpers/custom-element-scoping'
+import { styles } from './form-control-footer.styles'
+
+import '../../components/badge'
+import '../../components/icon/icons/triangle-exclamation'
+
+/**
+ * @element gds-form-control-footer
+ */
+@gdsCustomElement('gds-form-control-footer')
```

Comment:
I would suggest to name this to `gds-field-footer` and same pattern for header, since it is a bit more clear on what it is, also in design perhaps could be a bit more easy to understand.

Reply:
I would argue that `form control` is the technically correct term that we should stick to in the context of forms.

For reference, W3C defines it as "An HTML form is a section of a document containing normal content, markup, special elements called [controls](https://www.w3.org/TR/html401/interact/forms.html#form-controls)"